packages/parity-electron/src/checkTime.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+const sntp = require('sntp');
+
+export const checkTime = () => {
+  return sntp.time().then(({t}) => ({drift: t}));
```

Comment:
Would you mind installing the prettier extension on your IDE? It'll allow everyone to have the exact same code format.

I'll add it in the tslint config on the js libs monorepo at some point too.